frontend/packages/dev-console/src/components/edit-application/edit-application-types.ts

```diff
@@ -0,0 +1,24 @@
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { FirehoseResult } from '@console/internal/components/utils';
+import { NormalizedBuilderImages } from '../../utils/imagestream-utils';
+
+export interface EditApplicationFormProps {
```

Comment:
These props should be co-located with the component. I don't see them being used anywhere else.